helmfile.d/40-realtime-dashboard.yaml

```diff
@@ -18,7 +18,7 @@ releases:
     set:
       - name: kafka_num_brokers
         value: {{ .Values.kafka_num_brokers }}
-      - name: jdbc.username
```

Comment:
I think I've seen this change before, was it in this component or JDBC connector?

Reply:
Not sure. I think it is `jdbc.user` in jdbc-connector but `jdbc.username` in   jdbc-connector-agg